src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -111,15 +112,34 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Nationality updatedNationality = editPersonDescriptor.getNationality().orElse(personToEdit.getNationality());
         TutorialGroup updatedTutorialGroup = editPersonDescriptor.getTutorialGroup()
                 .orElse(personToEdit.getTutorialGroup());
-        SocialHandle updatedSocialHandle = editPersonDescriptor.getSocialHandle()
-                .orElse(personToEdit.getSocialHandle());
         Gender updatedGender = editPersonDescriptor.getGender()
                 .orElse(personToEdit.getGender());
         Remark updatedRemark = editPersonDescriptor.getRemark().orElse(personToEdit.getRemark());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
+        Set<SocialHandle> updatedSocialHandles = updateSocialHandles(personToEdit, editPersonDescriptor);
```

Comment:
I understand where you are coming from, however it may not be practical to change it as this order is implemented throughout the codebase, including the argument order for Person. Changing the order could be time-intensive and could result in inconsistency as some parts may be missed. Therefore, given the limited time we have left, I feel that it may not be worth it as the time could be better spent on other places. The ordering could be fixed in v1.4 when have finalized everything have have the time to do so. For now, what I can do is to only change the order in all the message for the user.